assets/settings/reducers.js

```diff
@@ -26,6 +28,7 @@ const initialState = {
     activeQuery: null,
     selectedMenu: 'companies',
     services: [],
+    productsById: {},
```

Comment:
would be good imo if we put this into `products` namespace so we can combine reducers by settings app instead of having it all together